wire-runtime/src/test/proto-kotlin/com/squareup/wire/protos/person/Person.kt

```diff
@@ -0,0 +1,115 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source file: person.proto
+package com.squareup.wire.protos.person
+
+import com.squareup.wire.EnumAdapter
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import com.squareup.wire.WireEnum
+import com.squareup.wire.kotlin.UnkownFieldsBuilder
+import kotlin.Int
+import kotlin.String
+import kotlin.collections.List
+import okio.ByteString
+
+data class Person(
+    val name: String,
+    val id: Int,
+    val email: String? = null,
+    val phone: List<PhoneNumber>,
+    val unknownFields: ByteString = ByteString.EMPTY
+) {
+    object ADAPTER : ProtoAdapter<Person>(FieldEncoding.LENGTH_DELIMITED, Person::class.java) {
+        override fun encodedSize(value: Person): Int = ProtoAdapter.STRING.encodedSizeWithTag(1, value.name) + 
+        ProtoAdapter.INT32.encodedSizeWithTag(2, value.id) + 
```

Comment:
Bad indent. Report a KotlinPoet bug. It should either indent this or reject turning it into an expression function because the statement wraps to multiple lines.

Reply:
fixed the indent here and https://github.com/square/kotlinpoet/issues/415